integration-tests/docker/broker.conf

```diff
@@ -11,6 +11,7 @@ command=java
   -Ddruid.host=%(ENV_HOST_IP)s
   -Ddruid.zk.service.host=druid-zookeeper-kafka
   -Ddruid.processing.buffer.sizeBytes=25000000
+  -Ddruid.query.groupBy.maxOnDiskStorage=300000000
```

Comment:
Why was this change needed?

Reply:
Because the new integration tests I added needed more aggregation space than the 25MB of memory allocated by the previous line. I didn't want to allocate more memory (and therefore raise the memory requirements of integration tests) so instead I am configuring it to use disk for the spillover.